process-automation/latest/model/connectors/teams.md

```diff
@@ -16,32 +16,29 @@ The actions that can be executed using the Teams connector are:
 
 ## Configuration
 
-For the Teams connector to function specifc properties must be defined in the `application.properties` file of the Spring Boot application. By default, these properties are set to environment variables.
-
-The following is a set of standard properties that define the use of the connector:
-
-```bash
-teams.connector.client.id=${TEAMS_CLIENT_ID}
-teams.connector.client.secret=${TEAMS_CLIENT_SECRET}
-teams.connector.scope=${TEAMS_SCOPE}
-teams.connector.username=${TEAMS_USERNAME}
-teams.connector.password=${TEAMS_PASSWORD}
-teams.connector.tenant=${TEAMS_TENANT}
-```
-
-The connector uses a stream mechanism to send and receive information between Process Services and the connector. For this to occur you need to configure the following properties so the connector can be identified:
-
-```bash
-spring.cloud.stream.bindings.createTeamsChannel.destination=${teams.connector.name}${spring.application.name}.CREATE_CHANNEL
-spring.cloud.stream.bindings.createTeamsChannel.contentType=application/json
-spring.cloud.stream.bindings.createTeamsChannel.group=${CONSUMER_GROUP:${spring.application.name}}
-spring.cloud.stream.bindings.getTeams.destination=${teams.connector.name}${spring.application.name}.GET_TEAMS
-spring.cloud.stream.bindings.getTeams.contentType=application/json
-spring.cloud.stream.bindings.getTeams.group=${CONSUMER_GROUP:${spring.application.name}}
-spring.cloud.stream.bindings.getChannels.destination=${teams.connector.name}${spring.application.name}.GET_CHANNELS
-spring.cloud.stream.bindings.getChannels.contentType=application/json
-spring.cloud.stream.bindings.getChannels.group=${CONSUMER_GROUP:${spring.application.name}}
-```
+### Teams connector configuration parameters
+
+The configuration parameters for the Teams connector are:
+
+| Parameter | Description |
+|-----------|-------------|
+| TEAMS_CLIENT_ID | *Required.* The client identifier to be used for authentication. |
+| TEAMS_CLIENT_SECRET | *Required.* The client secret to be used for authentication. |
+| TEAMS_USERNAME | *Required.* The MS Teams user to impersonate `in:wq` the connector. |
```

Comment:
I don't understand this sentence.

Reply:
@pmartinezga  what does this mean? | TEAMS_USERNAME | *Required.* The MS Teams user to impersonate `in:wq` the connector. |

Thanks.

Reply:
@oliveralfresco2018 , I checked with Pablo. This should read "in the connector" (instead of "in:wq the connector")

Reply:
Thats awesome. Thanks.